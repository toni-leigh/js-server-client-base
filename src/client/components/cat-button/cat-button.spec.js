import React from 'react';
import { shallow } from 'enzyme';
import CatButton from './cat-button';

describe('<CatButton />', () => {
  let catButton;

  beforeAll(() => {
    catButton = shallow(
      <CatButton />
    );
  });

  test('basic render', () => {
    expect(catButton).toMatchSnapshot();
  });
});
