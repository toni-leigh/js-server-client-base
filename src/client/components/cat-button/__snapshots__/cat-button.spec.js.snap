// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatButton /> basic render 1`] = `
<button
  className="crardx-cat-button"
>
  Get cat!
</button>
`;
